install/kubernetes/helm/istio/values.yaml

```diff
@@ -522,11 +522,10 @@ kiali:
       #   hosts:
       #     - kiali.local
   dashboard:
-    username: admin
-    # Default admin passphrase for kiali. Must be set during setup, and
-    # changed by overriding the secret
-    passphrase: admin
-
+    secretName: kiali
+    usernameKey: username
+    passphraseKey: passphrase
```

Comment:
Can we use `passwordKey`? So that we can keep consistent with Grafana keyword at https://github.com/istio/istio/pull/7719

Reply:
done

Reply:
Sorry folks for the churn.  PTO for 2 weeks and then catching up on backlog.  This PR looks good, however, some security professionals prefer the word "passphrase" to the word "password".

I'm going to go rescue some seagulls now: https://www.usenix.org/system/files/1401_08-12_mickens.pdf

Reply:
done